src/Components/Aspire.OpenAI/AspireOpenAIExtensions.cs

```diff
@@ -69,9 +69,11 @@ private static void AddOpenAIClient(
         ArgumentNullException.ThrowIfNull(builder);
 
         var configSection = builder.Configuration.GetSection(configurationSectionName);
+        var namedConfigSection = configSection.GetSection(connectionName);
 
         OpenAISettings settings = new();
         configSection.Bind(settings);
+        namedConfigSection.Bind(settings);
```

Comment:
We should update line 91 as well below to bind the ClientOptions to the named config section.